### PR TITLE
Delta lake walkthrough patch

### PR DESCRIPTION
**Pull request by syed-tw**
Creating the pull request to compare Delta Lake Walkthrough file between the master and `Delta lake walkthrough patch` branch

### CODE REVIEW

File: delta-lake-walkthrough/delta-lake-walkthrough.py [hunk 1]
```diff
@@ -204,15 +203,10 @@ def cleanup_paths_and_tables():
...
-# MAGIC %md
-# MAGIC Use the above output to create a table from the parquet location.
-
-# COMMAND ----------
-
 # MAGIC %sql
 # MAGIC CREATE TABLE loans_delta2
 # MAGIC USING delta
-# MAGIC AS SELECT * FROM parquet.`/FileStore/YOURUSERNAME/deltaDemo/loans_parquet/`
+# MAGIC AS SELECT * FROM parquet.`/FileStore/syedalimasroor.r/deltaDemo/loans_parquet/`
```
Comment (kelseymok) [inline delta-lake-walkthrough/delta-lake-walkthrough.py:209]: Can you change this back to `YOURUSERNAME` please?

File: delta-lake-walkthrough/delta-lake-walkthrough.py [hunk 2]
```diff
@@ -224,12 +218,13 @@ def cleanup_paths_and_tables():
...
-# MAGIC %md
-# MAGIC Use the above output to create a table from the parquet location.
+# MAGIC %md Use the above output to create a table from the parquet location.
 
 # COMMAND ----------
 
-# MAGIC %sql CONVERT TO DELTA parquet.`/FileStore/YOURUSERNAME/deltaDemo/loans_parquet/`
+# COMMAND ----------
+
+# MAGIC %sql CONVERT TO DELTA parquet.`/FileStore/syedalimasroor.r/deltaDemo/loans_parquet/`
```
Comment (kelseymok) [inline delta-lake-walkthrough/delta-lake-walkthrough.py:227]: Can you change this back to `YOURUSERNAME` please?

**Review by kelseymok — CHANGES_REQUESTED** [review]
There are a few places with your name in the code (marked)

File: delta-lake-walkthrough/delta-lake-walkthrough.py (hunk 2 above)
Comment (syed-tw) [reply delta-lake-walkthrough/delta-lake-walkthrough.py]: Changes are done now and commited. Can you please check now and merge the PR ?

**Review by kelseymok — APPROVED** [review]

**Pull request merged**